aspnetcore/data/ef-rp/complex-data-model.md

```diff
@@ -618,13 +625,11 @@ Examine the **CourseAssignment** table:
 
 ![CourseAssignment data in SSOX](complex-data-model/_static/ssox-ci-data.png)
 
-<a name="fk"></a>
-
-## Fixing foreign key constraints with legacy data
+<a name="applyexisting"></a>
 
-This section is optional.
+### Apply the migration to the existing database
 
-When migrations are run with existing data, there may be FK constraints that are not satisfied with the exiting data. With production data, steps must be taken to migrate the existing data. This section provides an example of fixing FK constraint violations. Don't make these code changes without a backup. Don't make these code changes if you completed the previous section and updated the database.
+When migrations are run with existing data, there may be FK constraints that are not satisfied with the existing data. With production data, steps must be taken to migrate the existing data. This section provides an example of fixing FK constraint violations. Don't make these code changes without a backup. Don't make these code changes if you completed the previous section and updated the database.
```

Comment:
Here I'd start off with an explanation that it's optional in case they missed it earlier, something like

> This section is optional. These steps work only if you skipped the preceding **Drop and re-create the database** section.